.github/workflows/build.yml

```
@@ -0,0 +1,187 @@
+        body: ${{ env.changelog }}
+        draft: false
+        prerelease: false
+    - name: Upload youtube-dl Unix binary
+      id: upload-release-asset
+      uses: actions/upload-release-asset@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        upload_url: ${{ steps.create_release.outputs.upload_url }}
+        asset_path: ./youtube-dl
+        asset_name: youtube-dl
+        asset_content_type: application/octet-stream
+    - name: Upload Source tar
+      uses: actions/upload-release-asset@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        upload_url: ${{ steps.create_release.outputs.upload_url }}
+        asset_path: ./youtube-dl.tar.gz
+        asset_name: youtube-dl-${{ steps.bump_version.outputs.ytdl_version }}.tar.gz
+        asset_content_type: application/gzip
+
+  build_windows:
+    runs-on: windows-2022
+    needs: build_unix
+    outputs:
+      sha256_win: ${{ steps.sha256_win.outputs.sha256_win }}
+      sha512_win: ${{ steps.sha512_win.outputs.sha512_win }}
+
+    steps:
+    - uses: actions/checkout@v2
+    # reason to choose 3.4: https://media.discordapp.net/attachments/807245652072857613/942409077701619742/unknown.png
+    - name: Set up Python 3.4
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.4'
+        architecture: 'x86'
+    - name: Install packages
+      # https://setuptools.pypa.io/en/latest/history.html#v44-0-0
+      # https://pypi.org/project/py2exe/0.9.2.2/
+      # https://pip.pypa.io/en/stable/news/#v19-2
+      # https://wheel.readthedocs.io/en/stable/news.html
+      run: python -m pip install --upgrade "pip<19.2" "setuptools<44" "wheel<0.34.0" py2exe==0.9.2.2
+    - name: Bump version
+      id: bump_version
+      env:
+        version_suffix: ${{ needs.build_unix.outputs.version_suffix }}
+      run: python devscripts/update-version.py ${{ env.version_suffix }}
+    # - name: Run PyInstaller Script
+    #   run: python -m PyInstaller --onefile --console --distpath dist/ -n youtube-dl youtube_dl\__main__.py
+    - name: Build EXE file
+      run: python setup.py py2exe
+    - name: Upload youtube-dl.exe Windows binary
+      id: upload-release-windows
+      uses: actions/upload-release-asset@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        upload_url: ${{ needs.build_unix.outputs.upload_url }}
+        asset_path: ./youtube-dl.exe
+        asset_name: youtube-dl.exe
+        asset_content_type: application/vnd.microsoft.portable-executable
+    - name: Get SHA2-256SUMS for youtube-dl.exe
+      id: sha256_win
+      run: echo "::set-output name=sha256_win::$((Get-FileHash youtube-dl.exe -Algorithm SHA256).Hash.ToLower())"
+    - name: Get SHA2-512SUMS for youtube-dl.exe
+      id: sha512_win
+      run: echo "::set-output name=sha512_win::$((Get-FileHash youtube-dl.exe -Algorithm SHA512).Hash.ToLower())"
```

```suggestion
    - name: Get SHA2-256SUMS for youtube-dl.exe
      id: sha256_win
      run: echo "sha256_win=$((Get-FileHash youtube-dl.exe -Algorithm SHA256).Hash.ToLower())" >> $GITHUB_OUTPUT
    - name: Get SHA2-512SUMS for youtube-dl.exe
      id: sha512_win
      run: echo "sha512_win=$((Get-FileHash youtube-dl.exe -Algorithm SHA512).Hash.ToLower())" >> $GITHUB_OUTPUT
```

`set-output` is depreciated